documentation/puppet-api/v3/environment_classes.md

```diff
@@ -376,7 +376,7 @@ the class information in later queries to the environment classes endpoint. To c
 entries on the server, do one of the following:
 
 1.  Call the
-    [`environment-cache` API endpoint](/puppet-server/latest/admin-api/v1/environment-cache.html).
+    [environment cache API][].
```

Comment:
@rlinehan Considering the following paragraph, it's probably best to revert the formatting and use the reference link as the target to fix the link:

```
[`environment-cache` API endpoint][environment cache API].